.evergreen/integration-tests.sh

```diff
@@ -155,15 +146,26 @@ case "$OS" in
       ;;
 esac
 
-sleep 15
-echo "Checking that mongo-orchestration is running"
-curl http://localhost:8889/ -sS --max-time 120 --fail | python -m json.tool
-
+echo "Waiting for mongo-orchestration to start..."
+wait_for_mongo_orchestration() {
+   for i in $(seq 300); do
+      # Exit code 7: "Failed to connect to host".
+      if curl -s "localhost:$1"; test $? -ne 7; then
+         return 0
+      else
+         sleep 1
+      fi
+   done
+   echo "Could not detect mongo-orchestration on port $1"
+   return 1
+}
+wait_for_mongo_orchestration 8889
+echo "Waiting for mongo-orchestration to start... done."
+
+curl http://localhost:8889/ -sS --fail
```

Comment:
```suggestion
```

Suggest removing, as the last `curl` command in `wait_for_mongo_orchestration` will have the same output.